static/src/javascripts/projects/commercial/modules/header-bidding/prebid/prebid.js

```diff
@@ -150,12 +160,25 @@ const requestBids = (
             () =>
                 new Promise(resolve => {
                     window.pbjs.que.push(() => {
+                        // TODO: Replace with commercial core's API
+                        recordFirstPrebidStarted();
+                        const adUnitsCodes = adUnits.map(adUnit => stripDfpAdPrefixFrom(adUnit.code));
+                        if (adUnitsCodes.indexOf('top-above-nav') !== -1) {
+                            markTime(`Prebid Started for Top Above Nav (${adUnitsCodes})`);
```

Comment:
We only want this to fire once. The first time the slot does this. Not again when the slot is refreshed.

Reply:
Yeah this would capture the refreshes as well - we could wrap it into a "once" function but I will wait to integrate commercial core timers API